sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -381,6 +381,22 @@ public void testDropUserStmtBuilder() {
     }
 
     @Test
+    public void testGrantPrivilegeStmtBuilder() {
+        printStatement("grant DML To \"Günter\"");
+        printStatement("grant DQL, DDL to root");
+        printStatement("grant ALL PRIVILEGES to wolfie");
+        printStatement("grant ALL PRIVILEGES to wolfie, anna");
```

Comment:
missing test without PRIVILEGES only with ALL
missing test with multiple privileges and multiple users